File: Lab_1/src/Circle.java
public class Circle extends Figure {
    private final int x, y, radius;

    public Circle(int x, int y, int radius) {
        this.x = x;
        this.y = y;
        this.radius = radius;
    }

    @Override
    public void draw() {
        System.out.printf("Circle(x = %d, y = %d, radius = %d)\n", x, y, radius);
    }
}

File: Lab_1/Circle.cs
public class Circle : Figure
{
    double x, y, r;
    public Circle(double x, double y, double r)
    {
        this.x = x;
        this.y = y;
        this.r = r;
    }

    public override void drawFigure()
    {
        Console.WriteLine($"Circle({x}, {y}, {r})");
    }
}